src/Core/src/Platform/Standard/EditorExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.Maui
+{
+	public static class EditorExtensions
```

Comment:
We don't need these anymore. So this can be deleted.

Reply:
Deleted.